cmssw/run.sh

```diff
@@ -14,7 +14,6 @@ cd $CMSSW_VERSION/src
 eval `scramv1 runtime -sh`
 git cms-init --upstream-only
 git remote add SegLink https://github.com/SegmentLinking/cmssw.git
-git sparse-checkout set .gitignore .clang-format .clangtidy
```

Comment:
the alternative is to have `git sparse-checkout set /.gitignore /.clang-format /.clangtidy`